### PR TITLE
[libra-node] remove the disable logging flag

### PR DESCRIPTION
**Pull request by davidiw**
You can disable logging via config and env, why have yet another means
to do it?

### CODE REVIEW

File: testsuite/libra-swarm/src/swarm.rs [hunk 1]
```diff
@@ -74,14 +73,10 @@ impl LibraNode {
...
-        if disable_logging {
-            node_command.arg("-d");
-        } else {
-            node_command.env("STRUCT_LOG_FILE", struct_log_path);
-        }
         node_command
             .stdout(log_file.try_clone()?)
-            .stderr(log_file.try_clone()?);
+            .stderr(log_file.try_clone()?)
+            .env("STRUCT_LOG_FILE", struct_log_path);
```
Comment (bmwill) [inline testsuite/libra-swarm/src/swarm.rs:79]: We don't even respect this env variable anymore...I think?

**Review by bmwill — APPROVED** [review]

File: testsuite/libra-swarm/src/swarm.rs (hunk 1 above)
Comment (davidiw) [reply testsuite/libra-swarm/src/swarm.rs:79]: more to delete!

**Comment by bmwill** [comment]
Looks like there's just a single lint that needs to be taken care of

**Pull request closed without merging**